Can't install marytts on Ubuntu - TESTS fail

Hi you guys,
I'm brand new in the field and I've been trying to install marytts on Ubuntu 16.04 
Everything was fine until I bumped into this annoying error while running "mvn install". 
I have yet to figure out how to fix it, so please take a look and help me with this.
Thanks in advance for your help! 

```
mhn@mhn:/voice/sources/marytts$ JAVA_HOME=/voice/soft/java /voice/soft/maven/bin/mvn -X install
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T23:41:47+07:00)
Maven home: /voice/soft/maven
Java version: 1.8.0_112, vendor: Oracle Corporation
Java home: /voice/soft/java/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.4.0-45-generic", arch: "amd64", family: "unix"


-------------------------------------------------------
 T E S T S
-------------------------------------------------------
[DEBUG] boot classpath:  /home/hanguyen/.m2/repository/org/apache/maven/surefire/surefire-booter/2.19.1/surefire-booter-2.19.1.jar  /home/hanguyen/.m2/repository/org/apache/maven/surefire/surefire-api/2.19.1/surefire-api-2.19.1.jar  /voice/sources/marytts/marytts-common/target/test-classes  /voice/sources/marytts/marytts-common/target/classes  /home/hanguyen/.m2/repository/commons-lang/commons-lang/2.6/commons-lang-2.6.jar  /home/hanguyen/.m2/repository/commons-io/commons-io/2.5/commons-io-2.5.jar  /home/hanguyen/.m2/repository/junit/junit/4.12/junit-4.12.jar  /home/hanguyen/.m2/repository/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar  /home/hanguyen/.m2/repository/log4j/log4j/1.2.16/log4j-1.2.16.jar  /home/hanguyen/.m2/repository/com/twmacinta/fast-md5/2.7.1/fast-md5-2.7.1.jar  /home/hanguyen/.m2/repository/org/codehaus/groovy/groovy-all/2.4.5/groovy-all-2.4.5.jar  /home/hanguyen/.m2/repository/org/testng/testng/6.9.11/testng-6.9.11.jar  /home/hanguyen/.m2/repository/org/beanshell/bsh/2.0b4/bsh-2.0b4.jar  /home/hanguyen/.m2/repository/com/beust/jcommander/1.48/jcommander-1.48.jar  /home/hanguyen/.m2/repository/org/apache/ant/ant/1.7.0/ant-1.7.0.jar  /home/hanguyen/.m2/repository/org/apache/ant/ant-launcher/1.7.0/ant-launcher-1.7.0.jar  /home/hanguyen/.m2/repository/org/apache/maven/surefire/surefire-testng-utils/2.19.1/surefire-testng-utils-2.19.1.jar  /home/hanguyen/.m2/repository/org/apache/maven/surefire/surefire-grouper/2.19.1/surefire-grouper-2.19.1.jar  /home/hanguyen/.m2/repository/org/apache/maven/surefire/surefire-testng/2.19.1/surefire-testng-2.19.1.jar  /home/hanguyen/.m2/repository/org/apache/maven/surefire/common-java5/2.19.1/common-java5-2.19.1.jar
[DEBUG] boot(compact) classpath:  surefire-booter-2.19.1.jar  surefire-api-2.19.1.jar  test-classes  classes  commons-lang-2.6.jar  commons-io-2.5.jar  junit-4.12.jar  hamcrest-core-1.3.jar  log4j-1.2.16.jar  fast-md5-2.7.1.jar  groovy-all-2.4.5.jar  testng-6.9.11.jar  bsh-2.0b4.jar  jcommander-1.48.jar  ant-1.7.0.jar  ant-launcher-1.7.0.jar  surefire-testng-utils-2.19.1.jar  surefire-grouper-2.19.1.jar  surefire-testng-2.19.1.jar  common-java5-2.19.1.jar
Forking command line: /bin/sh -c cd /voice/sources/marytts/marytts-common && /voice/soft/java/jre/bin/java -Xms256m -Xmx512m -XX:MaxPermSize=128m -jar /voice/sources/marytts/marytts-common/target/surefire/surefirebooter2491858978837272470.jar /voice/sources/marytts/marytts-common/target/surefire/surefire1145511622047945376tmp /voice/sources/marytts/marytts-common/target/surefire/surefire_04616053325804941999tmp
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=128m; support was removed in 8.0
Running TestSuite
Configuring TestNG with: TestNG60Configurator
Test version expected null
Tests run: 84, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.148 sec <<< FAILURE! - in TestSuite
testVersion(marytts.VersionTest)  Time elapsed: 0.012 sec  <<< FAILURE!
java.lang.AssertionError: expected object to not be null
	at marytts.VersionTest.testVersion(VersionTest.java:12)


Results :

Failed tests: 
  VersionTest.testVersion:12 expected object to not be null

Tests run: 84, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] MaryTTS ............................................ SUCCESS [  2.195 s]
[INFO] marytts-common ..................................... FAILURE [  5.851 s]
[INFO] marytts-signalproc ................................. SKIPPED
[INFO] marytts-runtime .................................... SKIPPED
[INFO] marytts-lang-de .................................... SKIPPED
[INFO] marytts-lang-en .................................... SKIPPED
[INFO] marytts-lang-te .................................... SKIPPED
[INFO] marytts-lang-tr .................................... SKIPPED
[INFO] marytts-lang-ru .................................... SKIPPED
[INFO] marytts-lang-it .................................... SKIPPED
[INFO] marytts-lang-fr .................................... SKIPPED
[INFO] marytts-lang-sv .................................... SKIPPED
[INFO] marytts-lang-lb .................................... SKIPPED
[INFO] marytts-languages .................................. SKIPPED
[INFO] voice-cmu-slt-hsmm ................................. SKIPPED
[INFO] marytts-client ..................................... SKIPPED
[INFO] marytts-builder .................................... SKIPPED
[INFO] marytts-redstart ................................... SKIPPED
[INFO] marytts-transcription .............................. SKIPPED
[INFO] marytts-assembly-builder ........................... SKIPPED
[INFO] marytts-assembly-runtime ........................... SKIPPED
[INFO] marytts-assembly ................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 8.590 s
[INFO] Finished at: 2016-10-26T15:21:22+07:00
[INFO] Final Memory: 21M/208M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project marytts-common: There are test failures.
[ERROR] 
[ERROR] Please refer to /voice/sources/marytts/marytts-common/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project marytts-common: There are test failures.

Please refer to /voice/sources/marytts/marytts-common/target/surefire-reports for the individual test results.
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoFailureException: There are test failures.

Please refer to /voice/sources/marytts/marytts-common/target/surefire-reports for the individual test results.
	at org.apache.maven.plugin.surefire.SurefireHelper.reportExecution(SurefireHelper.java:91)
	at org.apache.maven.plugin.surefire.SurefirePlugin.handleSummary(SurefirePlugin.java:320)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:892)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:755)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 20 more
[ERROR] 
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :marytts-common
```

--- a/marytts-common/pom.xml
+++ b/marytts-common/pom.xml
@@ -56,5 +56,16 @@
 				<filtering>true</filtering>
 			</resource>
 		</resources>
+
+		<plugins>
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<configuration>
+					<systemPropertyVariables>
+						<version>${project.version}</version>
+					</systemPropertyVariables>
+				</configuration>
+			</plugin>
+		</plugins>
 	</build>
 </project>

